fastlane/Fastfile

```diff
@@ -253,6 +253,33 @@ platform :android do
       sh("git clean -fd")
     end
   end
+
+  desc "Runs Maestro UI tests"
+  lane :run_maestro_ui_tests do
+    emulator_name = ENV["MAESTRO_EMULATOR_NAME"] || UI.input("Name of the emulator you want to use? Note that it will be wiped on every run.")
+    google_email = ENV["MAESTRO_GOOGLE_EMAIL"] || UI.input("Email of the google account to use for tests?")
+    google_password = ENV["MAESTRO_GOOGLE_PASSWORD"] || UI.input("Password of the google account to use for tests?")
+
+    UI.message("Make sure 'emulator', 'adb' and 'maestro' are in your PATH.")
+
+    UI.message("Initializing emulator")
+    initialize_emulator(emulator_name)
+
+    UI.message("Installing purchase-tester")
+    Dir.chdir(get_root_folder) do
+      # Install purchase-tester in emulator
+      sh("./gradlew examples:purchase-tester:installLatestDependenciesDebug")
+    end
+
+    UI.message("Running tests")
+    run_maestro_play_store_login_flow(google_email, google_password, retry_count: 2)
```

Comment:
I added a simple retry mechanism since this is highly unreliable, and there are many weird issues that happens randomly. Once we add this to CI, the ideal would be to save screenshots of the emulator on the place where the test failed, to see what status it got to... But not for this PR.